docs/en/operation/Scalability-Tuning.md

```diff
@@ -8,20 +8,114 @@ priority: 9
 
 Alluxio is a scalable distributed file system designed to handle many workers within a single
 cluster.
-Several parameters can be tuned to prevent the Alluxio master from being overloaded.
-This page details the parameters to tune when scaling a cluster.
+This page details methods to recommend Alluxio node resource sizes. The section
+[Metrics to Monitor](#metrics-to-monitor) describes methods to monitor and estimate resources which
+influence the system's hardware requirements. The sections
+[Alluxio Master Configuration](#alluxio-master-configuration),
+[Alluxio Worker Configuration](#alluxio-worker-configuration), and
+[Alluxio Client Configuration](#alluxio-client-configuration) provide details on hardware and
+configuration tuning for large scale deployments.
 
 * Table of Contents
 {:toc}
 
+## Metrics To Monitor
+
+### Number of Files in Alluxio
+
+Files refers to files and directories. The number of files in Alluxio can be monitored through the
+metrics `Master.FilesCreated` plus `Master.DirectoriesCreated` minus `Master.PathsDeleted`. These
+metric will need to be aggregated by a third party metrics collector because the value will reset on
+each master restart or failover.
+
+The number of files in Alluxio impacts the following:
+* Size of heap required by the master - Each file takes approximately 1 - 2 kb. If RocksDB is used,
```

Comment:
why only if RocksDB is use? 
nit: Also please link to `en/operation/Metastore.html#rocksdb-metastore` if reference is needed

Reply:
Clarified, thanks.